tests/test_notifications.py

```diff
@@ -262,7 +261,7 @@ def test_node_subscriptions_and_backrefs_removed_when_node_is_deleted(self):
             subscription.save()
 
             s = getattr(project.creator, 'email_transactional', [])
-            assert_equal(len(s), 1)
+            assert_equal(len(s), 2)
```

Comment:
[Found it](https://github.com/CenterForOpenScience/modular-odm/blob/60ebd0c53b92e945ef8e87ff8bbbba9a16db8700/modularodm/storedobject.py#L230)!